patches/tModLoader/Terraria/ModLoader/UI/UICommon.cs

```diff
@@ -87,6 +88,25 @@ public static void DrawHoverStringInBounds(SpriteBatch spriteBatch, string text,
 		Terraria.UI.Chat.ChatManager.DrawColorCodedStringWithShadow(spriteBatch, FontAssets.MouseText.Value, text, vector, color, 0f, Vector2.Zero, Vector2.One);
 	}
 
+	public static void MouseText(string text, bool hoverBackground = false)
```

Comment:
Please add some simple documentation for this method.

Reply:
Is there any reason to use this method without passing `true` to `hoverBackground`? If so, are there places in tML that should use it? If not, should we just call this method `TooltipMouseText` or something? Given the relation to the `SettingsEnabled_OpaqueBoxBehindTooltips`

Reply:
I was more intending it to be for general MouseText, since I was planning on adding bounds and maximum width, but I didn't want to modify vanilla methods. I'll change the name and make it solely for MouseText with a background